src/app/project/FeaturesGrid.js

```diff
@@ -69,12 +74,14 @@ define([
                     renderExpando: true
                 },
                 action: 'Action',
-                subType: 'Treatment/Type'
+                subType: 'Treatment/Type',
+                opacity: {
+                    label: 'Opacity',
```

Comment:
Or without the header cell at all?
![image](https://cloud.githubusercontent.com/assets/1326248/8440457/82d248b4-1f2f-11e5-9c4f-d419b08fb85e.png)